### PR TITLE
Brings Back Old Brotherhood

### PR DESCRIPTION
**Pull request by waterwater1212**
## Description

Going to bring back the old Brotherhood roles, because I think they were better than whatever the fuck we've got going on right now. Desert Rose really fucked the code, anal with no lube. Still a WIP, don't merge. Also increased most BoS timers, hopefully this will stop mongrels from joining BoS to die to ghouls and leave BoS roles to robust fools who know what they're doing. Initiates will have to play a lot longer to unlock better roles, because remember, everything above Initiate is technically a leadership role according to the Chain-That-Binds. Knights have their t45 back, Paladins have t51.
## Motivation and Context

No more retarded "Ad Victoriam" BoS shit that autistic mentally-ill brain-leaking-out-of-ear DR coders have been adding with no thought. This'll hopefully keep newbies out of BoS and make sure you actually have to know what you're doing to play and lead.
## How Has This Been Tested?

Tested this all on a locally hosted thing. Spawning as Initiate Scribe & Initiate Knight works just fine.
## Changelog (necessary)
:cl:
add: Added Initiate Knight & Initiate Scribe start points
add: Mapped them into the bunker
del: Removed Knight-Captain
del: Removed Senior Paladin
del: Removed Initiate (replaced with separate Initiate Knight & Scribe roles)
del: Removed Sentinel
del: Removed Vertibird Pilot
tweak: Initiates have to play a lot longer, under the guidance of their appropriate higher-up (Knights for Initiate-Knights and Scribes for Initiate-Scribes) before they can start playing actually good roles.
tweak: Changed some BoS role descriptions
tweak: Changed leadership role exp_timers to not just be a couple hours.
/:cl:

### CODE REVIEW

**Comment by ErwinSnowden** [comment]
don't make changes to sunnydale, we've got mannypardo working on gutting the vault and adding more shit.

**Comment by waterwater1212** [comment]
Gonna work on it.

**Comment by Mannybrado** [comment]
you can fuck around with sunny right now. i'll tell everyone when i'm actually motivated to do the big shit i want to do for sunny

**Comment by waterwater1212** [comment]
Alright, pretty much done methinks. If there's anything else, please tell me and I'll get to it whenever.

File: code/game/objects/effects/landmarks.dm [hunk 1]
```diff
@@ -573,10 +573,14 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark/start/new_player)
...
+	name = "Initiate Knight"
 	icon_state = "Knight"
 
+/obj/effect/landmark/start/f13/initiatescribe
+	name = "Initiate Scribe"
+	icon_state = "Scribe"
+
 /obj/effect/landmark/start/f13/vertipilot
 	name = "Vertibird Pilot"
 	icon_state = "Knight"
```
Comment (Afya212) [inline code/game/objects/effects/landmarks.dm:586]: Forgot ta delete the old job landmarks.

Comment (waterwater1212) [reply code/game/objects/effects/landmarks.dm]: done bro

**Comment by waterwater1212** [comment]
STILL CONFLICT!!!! AAAAAHHHHHHH!!!!!

**Comment by waterwater1212** [comment]
nerds

**Pull request closed without merging**